ocis-pkg/version/version.go

```diff
@@ -1,18 +1,31 @@
 package version
 
 import (
-	"strconv"
 	"time"
 
 	"github.com/Masterminds/semver"
 )
 
 var (
-	// String gets defined by the build system.
-	String = "dev"
+	// String gets defined by the build system
+	String string
```

Comment:
Please... It might be out of scope, but can you name that differently? Please...